From 68f258112921e004ac9bb87ed7f21db1ae24a3e7 Mon Sep 17 00:00:00 2001
From: YueHaibing <yuehaibing@huawei.com>
Date: Tue, 17 Jul 2018 22:11:23 +0800
Subject: [PATCH 168/176] tipc: remove unused tipc_group_size

commit d81d25e66a0f218e7d6b6d81b2d57dacf8924195 upstream

After commit eb929a91b213 ("tipc: improve poll() for group member socket"),
it is no longer used.

Signed-off-by: YueHaibing <yuehaibing@huawei.com>
Acked-by: Jon Maloy <jon.maloy@ericsson.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Zhenbo Gao <zhenbo.gao@windriver.com>
---
 net/tipc/group.c |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/net/tipc/group.c b/net/tipc/group.c
index 1ed5446..97bf5fc 100644
--- a/net/tipc/group.c
+++ b/net/tipc/group.c
@@ -158,11 +158,6 @@ u32 tipc_group_exclude(struct tipc_group *grp)
 	return 0;
 }
 
-int tipc_group_size(struct tipc_group *grp)
-{
-	return grp->member_cnt;
-}
-
 struct tipc_group *tipc_group_create(struct net *net, u32 portid,
 				     struct tipc_group_req *mreq,
 				     bool *group_is_open)
-- 
1.7.9.5

